src/knockout/templates/question-imagepicker.html

```diff
@@ -22,7 +22,7 @@
                   <!-- /ko -->
                 </div>
                 <!-- ko if: question.showLabel -->
-                <span data-bind="css: question.koCss().itemText, attr: { title: item.locText.koRenderedHtml }">
+                <span data-bind="css: question.koCss().itemText">
```

Comment:
I an not sure that we have to remove title from image picker. Probaby it makes sense to have it.

Andrew

Reply:
We needn't a title here as well because text is visible - https://github.com/surveyjs/survey-library/blob/master/src/knockout/templates/question-imagepicker.html#L25-L28